scripts/sbom_scraper.sh

```diff
@@ -601,9 +601,9 @@ EOF
     if [ "${HTTP_STATUS:0:1}" != "2" ]
     then
         log "Upload failure ${HTTP_STATUS}"
+        cat "${TEMPDIR}/upload"
         exit 5
     fi
     log "Upload success: "
-    jq . "${TEMPDIR}/upload"
```

Comment:
Does removing this without also removing or changing the previous line make the output a bit confusing? ie "Upload success: " then nothing suggests something missing?

Reply:
Ah - will remove the ':'